envs/scIB-python.yml

```diff
@@ -5,508 +5,58 @@ channels:
-  - r-shiny=1.1.0=r351h6115d3f_0
-  - r-snow=0.4_3=r35h6115d3f_1001
-  - r-sourcetools=0.1.7=r351h29659fb_0
-  - r-spatial=7.3_11=r351hd10c6a6_4
-  - r-squarem=2017.10_1=r351h6115d3f_0
-  - r-stringi=1.2.4=r351h29659fb_0
-  - r-stringr=1.3.1=r351h6115d3f_0
-  - r-survival=2.42_6=r351h96ca727_0
-  - r-sys=3.2=r35hcdcec82_1
-  - r-testthat=2.0.0=r351h29659fb_0
-  - r-tibble=1.4.2=r351h96ca727_0
-  - r-tidyr=0.8.1=r351h29659fb_0
-  - r-tidyselect=0.2.4=r351h29659fb_0
-  - r-tidyverse=1.2.1=r351h6115d3f_0
-  - r-timedate=3043.102=r351h6115d3f_0
-  - r-tinytex=0.6=r351h6115d3f_0
-  - r-tsne=0.1_3=r35_1002
-  - r-ttr=0.23_3=r351ha65eedd_0
-  - r-usethis=1.5.1=r35h6115d3f_1
-  - r-utf8=1.1.4=r351h96ca727_0
-  - r-uuid=0.1_2=r351h96ca727_4
-  - r-vctrs=0.1.0=r351h96ca727_1000
-  - r-vipor=0.4.5=r35h6115d3f_1002
-  - r-viridis=0.5.1=r351h6115d3f_0
-  - r-viridislite=0.3.0=r351h6115d3f_0
-  - r-webshot=0.5.0=r351h6115d3f_0
-  - r-whisker=0.3_2=r351hf348343_4
-  - r-withr=2.1.2=r351h6115d3f_0
-  - r-xfun=0.3=r351h6115d3f_0
-  - r-xml2=1.2.0=r351h29659fb_0
-  - r-xopen=1.0.0=r35h6115d3f_1001
-  - r-xtable=1.8_2=r351h6115d3f_0
-  - r-xts=0.11_0=r351h96ca727_0
-  - r-yaml=2.2.0=r351h96ca727_0
-  - r-zeallot=0.1.0=r351h6115d3f_0
-  - r-zoo=1.8_3=r351h96ca727_0
-  - readline=8.0=hf8c457e_0
-  - rhash=1.3.6=h14c3975_1001
-  - seaborn=0.9.0=py_1
-  - sed=4.7=h1bed415_1000
-  - send2trash=1.5.0=py_0
-  - setuptools=41.0.1=py37_0
-  - simplegeneric=0.8.1=py_1
-  - sip=4.19.8=py37hf484d3e_1000
-  - six=1.12.0=py37_1000
-  - sqlite=3.29.0=hcee41ef_0
-  - statsmodels=0.10.1=py37hc1659b7_0
-  - tensorboard=1.14.0=py37hf484d3e_0
-  - tensorflow=1.14.0=mkl_py37h45c423b_0
-  - tensorflow-base=1.14.0=mkl_py37h7ce6ba3_0
-  - tensorflow-estimator=1.14.0=py_0
-  - terminado=0.8.2=py37_0
-  - testpath=0.4.2=py_1001
-  - tk=8.6.9=hed695b0_1002
-  - tktable=2.10=h555a92e_1
-  - tornado=6.0.3=py37h516909a_0
-  - tqdm=4.32.2=py_0
-  - traitlets=4.3.2=py37_1000
-  - tzlocal=2.0.0=py_0
-  - umap-learn=0.3.9=py37_0
-  - wcwidth=0.1.7=py_1
-  - webencodings=0.5.1=py_1
-  - werkzeug=0.15.5=py_0
-  - wheel=0.33.4=py37_0
-  - widgetsnbextension=3.5.0=py37_0
-  - wrapt=1.11.2=py37h7b6447c_0
-  - xorg-kbproto=1.0.7=h14c3975_1002
-  - xorg-libice=1.0.10=h516909a_0
-  - xorg-libsm=1.2.3=h84519dc_1000
-  - xorg-libx11=1.6.8=h516909a_0
-  - xorg-libxau=1.0.9=h14c3975_0
-  - xorg-libxdmcp=1.1.3=h516909a_0
-  - xorg-libxext=1.3.4=h516909a_0
-  - xorg-libxrender=0.9.10=h516909a_1002
-  - xorg-renderproto=0.11.1=h14c3975_1002
-  - xorg-xextproto=7.3.0=h14c3975_1002
-  - xorg-xproto=7.0.31=h14c3975_1007
-  - xz=5.2.4=h14c3975_1001
-  - yaml=0.1.7=had09818_2
-  - zeromq=4.3.2=he1b5a44_2
-  - zipp=0.5.2=py_0
-  - zlib=1.2.11=h516909a_1005
-  - zstd=1.4.0=h3b9ef0a_0
+  - numpy
+  - pandas
+  - seaborn
+  - matplotlib
+  - scanpy==1.4.6
+  - scipy
+  - memory_profiler
+  - rpy2>=3
+  - anndata2ri
+  - scanorama
+  - bbknn==1.3.9
+  - libgcc-ng
+  - scikit-learn
+  - networkx
+  - r-base
+  - r-seurat
```

Comment:
also add version, please... or maybe it matters less here, as it's not the `R` env.